postgres/lib/session.json

```diff
@@ -0,0 +1,7 @@
+{
+  "$user.id": "CAP.APPLICATIONUSER",
+  "$user.locale": "CAP.LOCALE",
+  "$user.now": "CAP.NOW",
+  "$valid.from": "CAP.VALID_FROM",
+  "$valid.to": "CAP.VALID_TO"
+}
```

Comment:
We discussed to prefer all-lowercase names (especially for PostgreSQL) → @agoerler did you check whether that is possible without breaking changes? I.e. if `current_setting` is case-insensitive?

Reply:
It is case insensitive and I will adjust accordingly